app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -713,7 +713,8 @@ public boolean onLongClick(View view) {
                 List<String> urlList = extractLinks(model.getContent());
                 StringBuilder url = new StringBuilder(urlList.get(0));
                 StringBuilder http = new StringBuilder("http://");
-                if (!(url.toString().startsWith(http.toString()) || url.toString().startsWith(http.toString()))) {
```

Comment:
@mayank408 That was a blunder. It was changed in your last PR #726. Please be a little careful and test the app thoroughly before sending a PR. Single line can make big changes. I also missed this while reviewing. Apologies for that. 
Lesson for now : Test the app thoroughly before sending a PR and review more carefully.